Naming/SubdirDirectoryPropertyNamer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Vich\UploaderBundle\Naming;
+
+use Vich\UploaderBundle\Mapping\PropertyMapping;
+use Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException;
+use Vich\UploaderBundle\Exception\NameGenerationException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
```

Comment:
What's the importance of sorting use declaration ? (quickly search on the web but can't find the information)